src/components/TransferForm/SetPaymailButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react'
+
+import { Button } from '@/components/Button'
+import SendIcon from '@mui/icons-material/Send'
+import { emitSetPaymailEvent } from './setPaymailEvent'
+import { sizes } from '@/styles'
+import { ButtonVariants } from '../Button/Button.styles'
+
+type SetPaymailButtonProps = {
+  paymail: string
+  variant?: ButtonVariants
+  onClick?: () => void
+}
+
+const customStyles = {
+  display: 'inline-flex',
+  padding: sizes(2),
+  marginLeft: sizes(2),
+}
+
+export const SetPaymailButton: FC<SetPaymailButtonProps> = ({ paymail, variant, onClick }) => {
```

Comment:
I've created a method for setting a paymail string into `TransferForm::PaymailInput` from any place using this `SetPaymailButton`. I've added it into transaction details and for every row in the `/Contacts` view.

Additionally, to get user's attention, I've implemented a `highlight-animation`.
![setPaymailButton2](https://github.com/bitcoin-sv/spv-wallet-web-frontend/assets/152964795/e79e2bae-217e-4a8f-9e98-0e7eae96da59)